packages/queued-request-controller/src/QueuedRequestController.ts

```diff
@@ -162,6 +165,8 @@ export class QueuedRequestController extends BaseController<
    */
   #showApprovalRequest: () => void;
 
+  #networkClientIdOfCurrentBatch?: NetworkClientId;
```

Comment:
Is it okay if we move this next to `originOfCurrentBatch` and add a small little comment description?

Reply:
done here: https://github.com/MetaMask/core/pull/4718/commits/2603f2530bddf9b1301d37cad319fdd93e5b8a28